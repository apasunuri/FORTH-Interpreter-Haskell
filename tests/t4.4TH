Result  is  34 46.2 8 / * CONCAT3 .  