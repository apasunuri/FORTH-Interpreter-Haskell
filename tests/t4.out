FORTH.exe: Wrong Data Types